ansible/roles/ml-analytics-service/templates/config.j2

```diff
@@ -28,7 +28,7 @@ evidence_base_url = {{ ml_analytics_evidence_base_url }}
 # Mongo  url
 #---------------
 
-url = mongodb://{{ ml_analytics_mongodb_url }}
+mongo_url = mongodb://{{ ml_analytics_mongodb_url }}
```

Comment:
What is the default value?

Reply:
This is configuration file for service. In code it is referring to mongo_url so updated the syntax which missed here. 
https://github.com/Sunbird-Ed/ml-analytics-service/blob/9be74f5a849c1d5d82408d12a00a6b8e777c8e0d/observations/py_observation_evidence_streaming.py#L67